Source/diablo.cpp

```diff
@@ -701,7 +701,7 @@ void GameEventHandler(uint32_t uMsg, int32_t wParam, int32_t lParam)
 		ReleaseKey(wParam);
 		return;
 	case DVL_WM_CHAR:
-		PressChar(wParam);
+		PressChar((char)wParam);

```

Comment:
```suggestion
		PressChar(static_cast<char>(wParam));
```

c++ style casts

Reply:
I didn't investigate if this is the proper type or place (just pushed it back from where it was definitely wrong), so keeping it as C style to make it clear that it needs more work